libs/utils/src/JobSystem.cpp

```diff
@@ -14,23 +14,21 @@
  * limitations under the License.
  */
 
-// Note: The overhead of SYSTRACE_TAG_JOBSYSTEM is not negligible especially with parallel_for().
-#ifndef SYSTRACE_TAG
-//#define SYSTRACE_TAG SYSTRACE_TAG_JOBSYSTEM
-#define SYSTRACE_TAG SYSTRACE_TAG_NEVER
-#endif
+// Note: The overhead of TRACING is not negligible especially with parallel_for().
+#define FILAMENT_TRACING_ENABLED false
```

Comment:
just curious, the choice of `false` over `0` is intentional?

Reply:
not really, does it matter?